packages/theme/components/CartSidebar.vue

```diff
@@ -0,0 +1,262 @@
+                    />
+                  </div>
+                </template>
+              </SfCollectedProduct>
+            </transition-group>
+          </div>
+        </div>
+        <div v-else key="empty-cart" class="empty-cart">
+          <div class="empty-cart__banner">
+            <SfImage
+              alt="Empty bag"
+              class="empty-cart__image"
+              src="/icons/empty-cart.svg"
+            />
+            <SfHeading
+              title="Your cart is empty"
+              :level="2"
+              class="empty-cart__heading"
+              description="Looks like you haven’t added any items to the bag yet. Start
+              shopping to fill it in."
+            />
+          </div>
+        </div>
+      </transition>
+      <template #content-bottom>
+        <transition name="sf-fade">
+          <div v-if="totalItems">
+            <SfProperty
+              name="Subtotal price"
+              class="sf-property--full-width sf-property--large my-cart__total-price"
+            >
+              <template #value>
+                <SfPrice
+                  :regular="$n(totals.subtotal, 'currency')"
+                />
+              </template>
+            </SfProperty>
+            <nuxt-link :to="localePath({ name: 'shipping' })">
+              <SfButton
+                v-e2e="'go-to-checkout-btn'"
+                class="sf-button--full-width color-secondary"
+                @click="toggleCartSidebar"
+              >
+                {{ $t('Go to checkout') }}
+              </SfButton>
+            </nuxt-link>
+          </div>
+          <div v-else>
+            <SfButton
+              class="sf-button--full-width color-primary"
+              @click="toggleCartSidebar"
+            >{{ $t('Go back shopping') }}</SfButton
+            >
+          </div>
+        </transition>
+      </template>
+    </SfSidebar>
+  </div>
+</template>
+<script>
+import {
+  SfSidebar,
+  SfHeading,
+  SfButton,
+  SfIcon,
+  SfProperty,
+  SfPrice,
+  SfCollectedProduct,
+  SfImage,
+  SfQuantitySelector
+} from '@storefront-ui/vue';
+import { computed } from '@vue/composition-api';
+import { useCart, useUser, cartGetters } from '@vue-storefront/vendure';
+import { useUiState } from '~/composables';
+
+export default {
+  name: 'Cart',
+  components: {
+    SfSidebar,
+    SfButton,
+    SfHeading,
+    SfIcon,
+    SfProperty,
+    SfPrice,
+    SfCollectedProduct,
+    SfImage,
+    SfQuantitySelector
+  },
+  setup() {
+    const { isCartSidebarOpen, toggleCartSidebar } = useUiState();
+    const { cart, removeItem, updateItemQty, load: loadCart, loading } = useCart();
+    const { isAuthenticated } = useUser();
+    const products = computed(() => cartGetters.getItems(cart.value));
+    const totals = computed(() => cartGetters.getTotals(cart.value));
+    const totalItems = computed(() => cartGetters.getTotalItems(cart.value));
+    loadCart();
```

Comment:
Shouldn't it be in onMounted or onSSR?

Reply:
That's strange, I copy/pasted it from core. Removed as load was already called from AppHeader